src/sentry/static/sentry/app/views/organizationGroupDetails/groupTagValues.tsx

```diff
@@ -118,7 +118,7 @@ class GroupTagValues extends AsyncComponent<
           </a>
           <DataExport
             payload={{
-              queryType: 'Issues-by-Tag',
+              queryType: ExportQueryType.IssuesByTag,
```

Comment:
Modifying the TS type as per my comment above makes this a little bit stricter too, which is nice.